.stylelintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "rules": {
+        "selector-class-pattern": [null]
```

Comment:
Reasoning behind the decision of disabling the `selector-class-pattern` rule:
We are loosely following the BEM naming convention and this rule expects classes to be named using kebab case. Installing a plugin to reinforce the BEM convention would mean we also need to update all the class names that are not using it and that is too much overhead for the current task.